From 65c31eb178d040f5ef96bf329a4633d6f2bf433d Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Fri, 24 Aug 2012 10:01:41 +0800
Subject: [PATCH 047/310] ENGR00219871 Camera: Last several seconds has no
 audio output of recorded video

Make the bdflush thread scheduled more frequently, which will make the
write() system call more smoothly for 1080P recording

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 imx6/etc/init.rc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index 71874de..4af4404 100644
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -231,7 +231,8 @@ on boot
     chmod 0664 /sys/module/lowmemorykiller/parameters/minfree
 
     # Tweak background writeout
-    write /proc/sys/vm/dirty_expire_centisecs 200
+    write /proc/sys/vm/dirty_expire_centisecs 50
+    write /proc/sys/vm/dirty_writeback_centisecs 100
     write /proc/sys/vm/dirty_background_ratio  5
 
     # Permissions for System Server and daemons.
-- 
1.8.0

